userguide/content/en/docs/Updating/Convert-site-to-module.md

```diff
@@ -171,32 +168,47 @@ hugo: cleaned module cache for "github.com/twbs/bootstrap"
 ```
 {{% /alert %}}
 
-### Clean up your repository
+## Clean up of your repository
 
-Since your site now uses Hugo Modules, your previously used `themes` directory can be removed: 
+Since your site now uses Hugo Modules, you can clean up your repo now.
+At the command prompt, change to the root directory of your existing site first.
```

Comment:
How about:

> Since your site now uses Hugo Modules, you can remove `docsy` from the `themes` directory, as instructed below. First, change to the root directory of your site:

Reply:
Works for me.

Reply:
Changed as requested.